### PR TITLE
ARXIVNG-899 stubs for license, policy, classification, secondary classification

### PR DESCRIPTION
**Pull request by JaimieMurdock**
The purpose of this ticket is to get stubs in place for several controllers and allow some immediate discussion of high level architectural decisions before finalizing.

What I see that can be improved:
- Almost all of these functions have a similar structure - what if we created a metafunction or decorator that wrapped the internal event-creation logic in all the code for action button selection.
- The application flow could be improved. @eawoods had a great suggestion for creating a list with the application flow and then fetching the next and previous button routes from that list.

For scoping, this PR is meant just to get the skeletons in for the `license`, `policy`, `classification`, and `crosslist` so that we can unblock ARXIVNG-874, 875, 876, and 877. It's meant to spur discussion, but not to necessitate the above changes - we just want to get to a working prototype and then have things we can refactor as we create tests next week.

### CODE REVIEW

File: submit/controllers/__init__.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
 from .verify_user import verify_user
 from .authorship import authorship
+from .license import license
```
Comment (erickpeirson) [inline submit/controllers/__init__.py]: ``license`` is a built-in...but a fairly useless one for our purposes, so probably not a big deal.

File: submit/controllers/classification.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+"
+Controller for classification actinos.
```
Comment (erickpeirson) [inline submit/controllers/classification.py]: Typo

File: submit/controllers/classification.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+"
```
Comment (eawoods) [inline submit/controllers/classification.py]: Needs triple quotes (here and subsequent files) to match.

Comment (DavidLFielding) [reply submit/controllers/classification.py]: I fixed and pushed these to GitHub a little while ago.

**Comment by erickpeirson** [comment]
I went ahead and did this so that we can move forward on other tickets. Thanks all!

**Pull request merged**